test/integration/render_task_test.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+require 'integration_test_helper'
+require 'kubernetes-deploy/render_task'
+
+class RenderTaskTest < KubernetesDeploy::IntegrationTest
+  def test_render_task
+    render = build_render_task(fixture_path('hello-cloud'))
+    fixture = 'configmap-data.yml'
+
+    stream = StringIO.new
```

Comment:
I'd like to keep it possible to run any test with `PRINT_LOGS=1` to see exactly how it would look to the user. This is the first time we've had as second output stream to deal with, so our current helpers (see in particular `configure_logger`, `logging_assertion` and `assert_logs_match`) aren't equipped to deal with it. One option would be to make new helpers just for this suite that similarly flip the output device between this new stream and stdout (and disable log assertions) as appropriate. However, if you see a nice way to adapt the existing helpers to handle this additional stream, that would be great. I'll keep thinking about this too.

Reply:
Ok, updated to add a `stdout_assertion` that works the same way with a different `StringIO` instance.